### ISSUE TITLE
cannot install the collection

### ISSUE THREAD

**Issue by msafarisc** [ansible-collections/community.sops#5]
##### SUMMARY
Seems like sops is not added to galaxy:

```
ansible-galaxy collection install community.sops
Process install dependency map
ERROR! Failed to find collection community.sops:*
```

##### ISSUE TYPE
- Installation issue

##### COMPONENT NAME
community.sops

##### ANSIBLE VERSION

```

ansible --version
ansible 2.9.10
  config file = None
  python version = 3.7.6 (default, Dec 30 2019, 19:38:26) [Clang 11.0.0 (clang-1100.0.33.16)]

```

##### OS / ENVIRONMENT
macOS

##### STEPS TO REPRODUCE
```
ansible-galaxy collection install community.sops
```

**Comment by msafarisc** [comment]
also https://galaxy.ansible.com/api/v2/collections/community/sops/ shows that sops is not added.
I wonder if there is a release process to do that.

**Comment by felixfontein** [comment]
As you noticed the collection has not yet been published. Right now you can only install it via git: `ansible-galaxy collection install https://github.com/ansible-collections/community.sops.git`.

@endorama do you have plans when to release a first version? I guess the README should be updated. Maybe you also want to add the `vars` plugin.

**Comment by endorama** [comment]
> @endorama do you have plans when to release a first version? I guess the README should be updated. Maybe you also want to add the vars plugin.

I do, I've been spending some times in adding tests for `vars` plugin (but I'm stuck). So I'm planning to release the first version with both plugins as they were in the Ansible PRs, without further modifications.

**Comment by endorama** [comment]
@felixfontein once #6 is merged, how shall I proceed with the release? I read [Publishing collections](https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#publishing-collections) but is not clear how that should play out for `community` collections (i.e. can I upload the collection under `community.sops`?)

**Comment by felixfontein** [comment]
@endorama publishing is done via Zuul and tagging: if you create and push a tag `0.1.0` (say), Zuul will automatically build the code tagged as version 0.1.0 to galaxy.

Before you do that, though, you should fill README.md with content, and add tags to galaxy.yml (these must be lower-case words, no spaces, dashes, dots etc. are allowed per tag). I already did some of the more technical things in #7 with a first tag in galaxy.yml.

**Comment by alexgeek** [comment]
> As you noticed the collection has not yet been published. Right now you can only install it via git: `ansible-galaxy collection install https://github.com/ansible-collections/community.sops.git`.

I get the following trying that:

```
$ ansible-galaxy collection install https://github.com/ansible-collections/community.sops.git
Process install dependency map
ERROR! Collection artifact at '/home/alex/.ansible/tmp/ansible-local-22202zh603qtf/tmpnbztl0wj/community.sopssqfn79dp.git' is not a valid tar file.
```

**Comment by felixfontein** [comment]
@alexgeek which version of Ansible(-base) are you using?

**Comment by alexgeek** [comment]
I'm using 2.9.11

**Comment by felixfontein** [comment]
Sorry, I only got around to check this out now... Yes, it doesn't work. But it is simple to fix: `ansible-galaxy collection install git+https://github.com/ansible-collections/community.sops.git` Unfortunately that does not work with Ansible 2.9.x, only with Ansible-base 2.10.0 (currently available as release candidate, Ansible 2.10.0 is still in alpha phase). Sorry :(

**Comment by alexgeek** [comment]
No problem I'm using it locally now (Y)
